### ISSUE TITLE
FalsePositive

### ISSUE THREAD

**Issue by sirian** [inspect-js/is-callable#50]
```javascript
class Foo{}
const Bar = Foo.bind(null);

assert(false === isCallable(Bar)); // fails, bound constructor is not callable
assert(false === isCallable(new Proxy(Foo, {})); // fails, constructor proxy is not callable
assert(false === isCallable(Proxy)); // fails, Proxy itself is not callable
```

**Comment by ljharb** [comment]
You already filed this and we discussed it in #30 :-)

**Comment by ExE-Boss** [comment]
```js
assert(false === isCallable(Proxy)); // fails, Proxy itself is not callable
```

That’s because `Proxy.toString()` returns:
```js
function Proxy(target, handler) { [native code] }
```

Instead of:
```js
class Proxy { [native code] }
```

**Comment by ljharb** [comment]
I'm confused; in which engine does the toString include argument names?

**Comment by ExE-Boss** [comment]
It’s allowed by the specification: <https://tc39.es/ecma262/#sec-function.prototype.tostring>.

**Comment by ljharb** [comment]
Sure it is, but which engines actually do it?
